sidechain/consensus/aura/src/verifier.rs

```diff
@@ -221,72 +221,124 @@ mod tests {
 		builders::parentchain_header_builder::ParentchainHeaderBuilder,
 		mock::onchain_mock::OnchainMock,
 	};
-	use its_test::sidechain_block_builder::SidechainBlockBuilder;
+	use its_primitives::types::SignedBlock;
+	use its_test::{
+		sidechain_block_builder::SidechainBlockBuilder,
+		sidechain_block_data_builder::SidechainBlockDataBuilder,
+		sidechain_header_builder::SidechainHeaderBuilder,
+	};
 	use sp_keyring::ed25519::Keyring;
-	use sp_runtime::{app_crypto::ed25519, testing::H256};
+	use sp_runtime::testing::H256;
 
 	fn assert_ancestry_mismatch_err<T: Debug>(result: Result<T, ConsensusError>) {
 		assert_matches!(result, Err(ConsensusError::BlockAncestryMismatch(_, _, _,)))
 	}
 
-	fn block2_builder(signer: ed25519::Pair, parent_hash: H256) -> SidechainBlockBuilder {
-		block1_builder(signer).with_parent_hash(parent_hash).with_number(2)
+	fn block1(signer: Keyring) -> SignedBlock {
+		let block_data = SidechainBlockDataBuilder::default()
+			.with_signer(signer.pair())
+			.with_timestamp(0)
+			.with_layer_one_head(default_header().hash())
+			.build();
+
+		let header = SidechainHeaderBuilder::default().with_block_number(1).build();
+
+		SidechainBlockBuilder::default()
+			.with_header(header)
+			.with_block_data(block_data)
+			.with_signer(signer.pair())
+			.build_signed()
```

Comment:
This seems to be a very repeating pattern (for block 1,2 and 3). I'm sure we can extract some common function here to not have as much code duplication?